.github/workflows/publish_release.yml

```diff
@@ -67,6 +67,9 @@ jobs:
           echo "swift_format_version=$SWIFT_FORMAT_VERSION" >> "$GITHUB_OUTPUT"
       - name: Checkout repository
         uses: actions/checkout@v4
+      - name: Check version matches
+        run: |
+          grep '"${{ steps.swift_format_version.outputs.swift_format_version }}"' Sources/swift-format/PrintVersion.swift
```

Comment:
I think we should also take into account the prerelease format, such as `"602.0.0-prerelease-$(date +'%Y-%m-%d')"` 🤔